#! /bin/sh /usr/share/dpatch/dpatch-run
## 02-stamp-h-in.dpatch by Philipp Benner <mail@philipp-benner.de>
##
## DP: Create stamp-h.in required to inhibit autoheader execution.

@DPATCH@
diff -urNad wmmemload-0.1.6~/stamp-h.in wmmemload-0.1.6/stamp-h.in
--- wmmemload-0.1.6~/stamp-h.in	1970-01-01 01:00:00.000000000 +0100
+++ wmmemload-0.1.6/stamp-h.in	2007-07-25 11:19:18.000000000 +0200
@@ -0,0 +1 @@
+timestamp
